R/utilities-building-block.R

```diff
@@ -0,0 +1,177 @@
+#' Set or add initial conditions to an existing Intitial Conditions building block.
+#'
+#' This functions allows adding or modifying initial condition entries.
+#'
+#' @param icBuildingBlock A `BuildingBlock` object of type `Initial Conditions`.
+#' The entries will be added to or set in this building block.
+#' @param quantityPaths A list of full paths of the quantities (usually molecules). Should contain
+#' all path elements and the molecule name, separated by `|`.
+#' @param quantityValues A list of values for the quantities. The length of this list should be equal to the length of `quantityPaths`.
```

Comment:
according to the comment below, `quantityValues `can be null if the `formulas `argument is not null